### ISSUE TITLE
xcparse should give better error when xcresult path leads to non-xcresult

### ISSUE THREAD

**Issue by Blackjacx** [ChargePoint/xcparse#40]
**Describe the bug**
I want to install xcparse using [Mint](https://github.com/yonaskolb/Mint) using the command `mint run ChargePoint/xcparse xcparse screenshots --test-run --model "$results_path" "$screens_path"`

But that doesn't work and gives the following output:
```
➜ mint run ChargePoint/xcparse xcparse screenshots --test-plan-config --model "/tmp" "/tmp"
🌱  Finding latest version of xcparse
🌱  xcparse 2.1.0 already installed
🌱  Running xcparse 2.1.0...
Error: Fatal error: Missing Info.plist: file /BuildRoot/Library/Caches/com.apple.xbs/Sources/XCResultKit/XCResultKit-15700/Sources/XCResultKit/RootInfoPlistParser.swift, line 52
```

### OS
![Screen Shot 2020-02-06 at 12 46 22](https://user-images.githubusercontent.com/794372/73934439-b97ae500-48de-11ea-9ef2-529e1ddd689f.png)

### xcparse Version
![Screen Shot 2020-02-06 at 12 47 16](https://user-images.githubusercontent.com/794372/73934484-d8797700-48de-11ea-9c09-93503bd009a6.png)

### Swift version [Run "swift --version"]
Apple Swift version 5.1.3 (swiftlang-1100.0.282.1 clang-1100.0.33.15)
Target: x86_64-apple-darwin19.2.0
 - XCResult version [Run "xcrun xcresulttool metadata get --path /path/to/xcresult"]

**To Reproduce**
Run the CMD above - it auto-resolves xcparse

**Expected behavior**
It should just work like installing it via brew.

**Comment by Blackjacx** [comment]
It actually wasn't an issue. Sorry for opening this ...

**Comment by abotkin-cpi** [comment]
Don't apologize - always good to open these issues so others can find it if they hit it! As I think you figured out, the xcresult path was likely set to something that wasn't an xcresult (like the tmp folder in the failing command above, if that isn't a redacted version).

This ticket still has a valid point that we should add protections in xcparse to do some basic sanity that the xcresult path actually has an xcresult there and not just throw it at xcresulttool blindly as it'll give that rather cryptic error when figuring out the manifest.

**Comment by Blackjacx** [comment]
Yeah this issue also occurs when I run a Homebrew installed xcparse. A sanity check would prevent this - true.

**Comment by SandraNPR** [comment]
I just ran into the same issue. The error was not descriptive enough. Luckily, with this post, I was able to figure out where I went wrong. Thanks!

**Comment by Blackjacx** [comment]
Glad to hear 🙃

**Comment by abotkin-cpi** [comment]
This should be addressed in 2.1.1. Thanks for the feedback @Blackjacx & @SandraNPR!
